torchvision/transforms/functional.py

```diff
@@ -948,12 +949,48 @@ def _get_inverse_affine_matrix(
     return matrix
 
 
+def _get_inverse_affine_matrix_tensor(
+    center: Tensor, angle: Tensor, translate: Tensor, scale: Tensor, shear: Tensor
+) -> Tensor:
+    output = torch.zeros(3, 3)
+
+    rot = angle * torch.pi / 180.0
+    shear_rad = shear * torch.pi / 180.0
+
+    m_center = torch.eye(3, 3)
+    m_center[:2, 2] = center
+
+    i_m_center = torch.eye(3, 3)
+    i_m_center[:2, 2] = -center
+
+    i_m_translate = torch.eye(3, 3)
+    i_m_translate[:2, 2] = -translate
+
+    # RSS without scaling
+    sx, sy = shear_rad[0], shear_rad[1]
+    a = torch.cos(rot - sy) / torch.cos(sy)
```

Comment:
just wondering, could here be division by zero?

Reply:
Yes, sy = pi/2 and same for sx=pi/2 with tan. IMO shear of ~90 degrees is too much distortion

Reply:
Maybe best to just protect against division by exact zero, or point it out in docs

Reply:
Docs however say that values between -180 and 180 are valid. I'm not sure how can we protect from zero devision if user requires 90 degree, like how to evaluate tan(pi/2) without getting infinity. Any ideas ?

Reply:
I don't think it should be solved in this issue since original non-tensor codes also have this problem. 
Maybe we can open a new issue about this. The potential solution might be just to modify the docs or clip within a range. 90 degree shearing seems meaningless.

Reply:
Yes, if it's not guarded against, the docs or a separate issue would be great.

Reply:
@vadimkantorov please create a new issue for this discussion. Thanks